spec/octokit/client/repositories_spec.rb

```diff
@@ -609,4 +609,38 @@
       expect(result).to be true
     end
   end # .disable_vulnerability_alerts
+
+  describe '.automated_security_fixes_enabled?', :vcr do
```

Comment:
Shouldn't this come with VCR updates? 

I'm not sure why I'm not seeing the prompt to approve the Actions test run; perhaps it has something to do with the new merge box UI. Either way, I'd expect these tests to need VCR updates to pass.